### ISSUE TITLE
`static_assert` errors in unit tests

### ISSUE THREAD

**Issue by stephanietsuei** [ucla-vision/xivo#32]
The matrix differential unit tests in `src/tests/test_rodrigues.cpp` currently do not compile. However, I think it is possible that the computed derivatives are correct despite these errors.

At the moment, these units tests are not compiled. To enable compilation, change the following lines in `src/CMakeLists.txt` from
```cmake
add_executable(unitTests_Rodrigues
               test/unittest_rodrigues.cpp)
```
to
```cmake
add_executable(unitTests_Rodrigues
               test/unittest_rodrigues.cpp
               test/test_rodrigues.cpp)
```

**Comment by agrawalparth10** [comment]
Based on my understanding, fixed the compilation error in #47. There are a few unit tests that fail. I think I will need some background to fix them.

**Comment by stephanietsuei** [comment]
Agreed. Opening up a new issue in #48 and closing this one.
